src/database/queries.ts

```diff
@@ -75,27 +84,63 @@ export const getVendor = async (uuid: Uuid): Promise<Vendor> => {
 };
 
 /**
- * Get all vouchers from the `vouchers` collection.
+ * Get all voucher ranges from the `voucher-ranges` collection.
+ *
+ * Returns an array of VoucherRange objects.
  */
-export const getAllVouchers = async (): Promise<Voucher[]> => {
+export const getAllVoucherRanges = async (): Promise<VoucherRange[]> => {
   try {
-    const dbQuery = query(voucherCollection);
+    const dbQuery = query(voucherRangeCollection);
     const querySnapshots = await getDocs(dbQuery);
+    return querySnapshots.docs.map(document => document.data() as VoucherRange);
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.warn('(getAllVoucherRanges)', e);
+    throw e;
+  }
+};
 
-    return querySnapshots.docs.map(document => document.data() as Voucher);
+/**
+ * Query the `voucher-ranges` collection.
+ *
+ * Return a VoucherRange if the given serialNumber is in any voucher range.
+ *
+ * Return `-1` if no such VoucherRange is found.
+ */
+const getVoucherRange = async (serialNumber: number) => {
```

Comment:
You should specify a return type -- `Promise<VoucherRange | null>`

I also think it's better to just return `null` rather than `-1`, so it's easy to check whether a result has been found by truthiness without having to remember particular error codes.